### PR TITLE
update fast-safe-stringify to avoid mutating input on toJSON

### PR DESCRIPTION
**Pull request by lallenlowe**
this version of fast-safe-stringify fixes an issue where it mutates input when calling the `toJSON` method of an object.

### CODE REVIEW

**Comment by lallenlowe** [comment]
good-squeeze PR for context:
https://github.com/hapijs/good-squeeze/pull/26

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "test-cov-html": "lab -m 5000 -r html -o coverage.html -La code"
   },
   "dependencies": {
-    "fast-safe-stringify": "1.1.3",
+    "fast-safe-stringify": "1.1.11",
```
Comment (arb) [inline package.json]: Let's make it `1.1.x`

Comment (lallenlowe) [inline package.json]: @arb kk, cool.

**Comment by lallenlowe** [comment]
@arb I made that change

**Pull request merged**